src/views/Login/LoginSelector.tsx

```diff
@@ -192,6 +192,13 @@ export default function LoginSelector({
         <Grid.Item full className="activate" as={Button} onClick={goToActivate}>
           Activate a Planet
         </Grid.Item>
+        <Grid.Item full className="l1-activation" as={Text}>
+          BitBlockBoom attendee?{' '}
```

Comment:
@nerveharp Since there are probably some other L1 invite cards floating around out threre, perhaps we could make this more generic? e.g., "Have an invite card with 8 words? "

Reply:
you probably mean 4 words (8 syllables)